returnn/tf/util/data.py

```diff
@@ -5039,16 +5039,18 @@ def get_batch_shape_dim_tags(self):
     return self.dim_tags
 
   @classmethod
-  def get_common_data(cls, sources, ignore_feature_dim=False, allow_broadcast_all_sources=NotSpecified):
+  def get_common_data(cls, sources, ignore_feature_dim=False, allow_broadcast_all_sources=NotSpecified, name=None):
```

Comment:
Is there any specific reason this was added, or just to make it nicer?

Reply:
Ah yeah, my rebase messed this up. This is #866. I'll remove it from here again.